Wbooks/Detail/Book Detail/BookDetailController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  BookDetailController.swift
+//  Wbooks
+//
+//  Created by Juan Martín Gordo on 21/04/2021.
+//
+
+import UIKit
+
+final class BookDetailController: UIViewController {
+
+    // MARK: Properties
+    private lazy var bookDetailView: BookDetailView = BookDetailView()
+    let viewModel: BookDetailViewModel
+    let errorTitle = NSLocalizedString("ALERT_BOX.TITLE", comment: "Title for the error alert box")
+    let rentErrorMessage = NSLocalizedString("ALERT_BOX.ERROR_MESSAGE", comment: "Message detailing an error in the alert box")
+    let errorMessage = NSLocalizedString("ALERT_BOX.BOOK_UNAVAILABLE_MESSAGE", comment: "Message detailing an error in the alert box")
+    let rentSuccessMessage = NSLocalizedString("ALERT_BOX.BOOK_RENTED", comment: "Message detailing an error in the alert box")
+    let errorDismiss = NSLocalizedString("ALERT_BOX.BUTTON", comment: "Text for the dismiss button on the alert box")
```

Comment:
Let's move these `NSLocalizedString` properties to the views (or controllers if it's something related to the navigation bar).

Reply:
Also, they don't need to be properties. They can be declared as local constants inside the methods they are used in.